extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventSupplementalFieldsMixin.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEventSupplementalFieldsMixin",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey Step Event Supplemental Fields Mixin",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Journey Step Event Supplemental Fields Mixin.",
```

Comment:
Since this is a field group, you will need to provide a class name in "meta:intendedToExtend" array; so that this field group is made available to the schemas that extend the class specified in intendedToExtend